SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -185,6 +185,12 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	void TrackTombstone(const Worker_EntityId EntityId);
 #endif
 
+	// If this flag is set to true standalone clients will not attempt to connect to a deployment automatically if connection config command line arguments exists.
+	UPROPERTY(Config)
+	bool bPreventAutoConnectWithCommandLineArgs;
```

Comment:
Does this need to be public?

Reply:
Also - was there a reason it's here, and not in our general gdk settings?

Reply:
It does not need to be public.

Not sure why it is not a setting either.